src/pages/API.js

```diff
@@ -0,0 +1,68 @@
+export default class API {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+    // constructor body
+  }
+
+  _checkResponse(res) {
```

Comment:
Great that you've implemented a  method for processing server response, so the code that checks the server response and converts it from JSON  is reused